libs/SalesforceSDKCommon/SalesforceSDKCommon/Classes/Logger/SFLogger.h

```diff
@@ -27,64 +25,81 @@
  WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-#define DD_LEGACY_MACROS 0
-#import <CocoaLumberjack/DDLog.h>
-#import "SFSDKFileLogger.h"
+#import <Foundation/Foundation.h>
+#import <OS/log.h>
+
+typedef NS_ENUM(NSUInteger, SFLogLevel){
+    SFLogLevelDefault =  OS_LOG_TYPE_DEFAULT,
```

Comment:
There is no warning level?

Reply:
Should we have a SFLogLevelWarning even if we end up mapping it to OS_LOG_TYPE_DEFAULT?

Reply:
@wmathurin Thats correct there isn't a warning equivalent. The Default sounds like the best choice for warning.